src/react-components/ory/user-auth-card.tsx

```diff
@@ -114,31 +115,33 @@ export type UserAuthCardProps = {
  * @returns JSX.Element
  */
 export const UserAuthCard = ({
-  flow,
   title,
   subtitle,
-  flowType,
   additionalProps,
   cardImage,
   onSubmit,
   includeScripts,
   className,
+  flow,
+  flowType,
 }: UserAuthCardProps): JSX.Element => {
+  // Safe, because we know that the props are of the correct type
+  const flowProps = { flow, flowType, additionalProps } as FlowProps
   const intl = useIntl()
 
   if (includeScripts) {
-    useScriptNodes({ nodes: flow.ui.nodes })
+    useScriptNodes({ nodes: flowProps.flow.ui.nodes })
   }
 
   if (!title) {
-    switch (flowType) {
+    switch (flowProps.flowType) {
       case "login":
-        if (flow.refresh) {
+        if (flowProps.flow.refresh) {
           title = intl.formatMessage({
```

Comment:
Type narrowing doesn't work on individual variables, that determine the type of the other. You need them to be part of an object. Then you can assert the type of one of its properties to narrow the whole type down.

Reply:
I think you need to explain that to me like I‘m 5 😅

Reply:
Oh I get it